src/helpers/git_helpers/branch_helpers.sh

```diff
@@ -14,6 +14,13 @@ function branch_needs_push {
 
 
 # Checkout a branch
+function checkout_branch {
+  local branch_name=$1
+  run_command "git checkout $branch_name"
+}
```

Comment:
This is already implemented in `checkout_helpers.sh`. The function is just called `checkout`